### PR TITLE
NextMillennium: Add `extraInfo` support and update `yaml`

### PR DESCRIPTION
**Pull request by marki1an**
As well update yaml file corresponding to this [PR](https://github.com/prebid/prebid-server/pull/2583)

### CODE REVIEW

**Review by KolomiietsSerhii — DISMISSED** [review]

File: src/main/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidder.java [hunk 1]
```diff
@@ -137,6 +139,15 @@ private static String formatSize(Integer w, Integer h) {
                 : null;
     }
 
+    private ObjectNode createImpExt(ExtRequestPrebid prebid) {
+        final ObjectNode impExt = mapper.mapper().createObjectNode();
+        impExt.set("prebid", mapper.mapper().valueToTree(prebid));
+        impExt.set("nextMillennium", CollectionUtils.isNotEmpty(nmmFlags)
+                ? mapper.mapper().valueToTree(nmmFlags)
+                : null);
```
Comment (CTMBNara) [inline src/main/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidder.java:147]:
```
        if (CollectionUtils.isNotEmpty(nmmFlags)) {
            impExt.putObject("nextMillennium")
                    .set("nmmFlags", mapper.mapper().valueToTree(nmmFlags));
        }
```

File: src/main/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidder.java [hunk 2]
```diff
@@ -109,10 +88,33 @@ private ExtImpNextMillennium convertExt(Imp imp, List<BidderError> errors) {
         return null;
     }
 
-    private List<HttpRequest<BidRequest>> makeRequests(BidRequest bidRequest, List<ExtImpNextMillennium> extImps) {
-        return extImps.stream()
-                .map(extImp -> makeHttpRequest(updateBidRequest(bidRequest, extImp)))
+    private BidRequest updateBidRequest(BidRequest bidRequest, ExtImpNextMillennium ext) {
+
```
Comment (CTMBNara) [inline src/main/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidder.java:92]: Please, remove empty line.

File: src/main/java/org/prebid/server/spring/config/bidder/NextMillenniumConfiguration.java [hunk 3]
```diff
@@ -23,19 +27,36 @@ public class NextMillenniumConfiguration {
...
 
-        return BidderDepsAssembler.forBidder(BIDDER_NAME)
+        return BidderDepsAssembler.<NextMillenniumConfigurationProperties>forBidder(BIDDER_NAME)
                 .withConfig(nextMillenniumConfigurationProperties)
                 .usersyncerCreator(UsersyncerCreator.create(externalUrl))
-                .bidderCreator(config -> new NextMillenniumBidder(config.getEndpoint(), mapper))
-                .assemble();
+                .bidderCreator(config -> {
+                    final ExtraInfo extraInfo = config.getExtraInfo();
+                    return new NextMillenniumBidder(config.getEndpoint(), mapper, extraInfo.getNmmFlags());
```
Comment (CTMBNara) [inline src/main/java/org/prebid/server/spring/config/bidder/NextMillenniumConfiguration.java:44]:
```
                .bidderCreator(config -> new NextMillenniumBidder(
                        config.getEndpoint(),
                        mapper,
                        config.getExtraInfo().getNmmFlags()))
```

**Review by CTMBNara — CHANGES_REQUESTED** [review]

**Review by CTMBNara — COMMENTED** [review]
Also remove `.filter(Objects::nonNull)` on line 70. This is redundant.

**Review by CTMBNara — APPROVED** [review]

**Pull request merged**